financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/SharedPartnerAuth.kt

```diff
@@ -291,6 +248,7 @@ private fun LoadedContent(
     when (authenticationStatus) {
         is Uninitialized,
         is Loading,
+        is Fail,
```

Comment:
Fail cases should not translate into error UIs anymore when the error is handled via navigation. It can be still used when there's specific in-context errors.

Reply:
100%!